compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/Window.desktop.kt

```diff
@@ -312,9 +510,13 @@ fun Window(
  * @param exitProcessOnExit should `exitProcess(0)` be called after the window is closed.
  * exitProcess speedup process exit (instant instead of 1-4sec).
  * If `false`, the execution of the function will be unblocked after application is exited
- * (when the last window is closed, and all [LaunchedEffect] are complete).
+ * (when the last window is closed, and all [LaunchedEffect]s are complete).
  * @param content Content of the window
  */
+@Deprecated(
+    level = DeprecationLevel.HIDDEN,
```

Comment:
Better to use `DeprecationLevel.WARNING` and provide `ReplaceWith`

Reply:
The reason I want HIDDEN is that with WARNING, code that specifies neither `undecorated` nor `decoration` becomes ambiguous:

<img width="954" alt="image" src="https://github.com/user-attachments/assets/1711b520-2cb1-4aef-ac53-d560786296d3">

Reply:
We can make `undecorated` required in the Deprecated overload, it will probably help.